docs/source/zero-code-change.rst

```diff
@@ -186,10 +186,10 @@ For Jupyter notebooks, use the following approach to turn on both:
 .. code-block::
 
    %load_ext cudf.pandas
-   from cuml.experimental.accel import install
+   from cuml.accel import install
```

Comment:
I believe

```
%load_ext cudf.pandas
%load_ext cuml.accel
```

_should_ work, and would be cleaner than telling the user to call `install` directly themselves.

If that doesn't work, it's probably due to double-modifying `rmm` settings, we could probably detect in `cuml` if rmm has been changed from the defaults and skip setting up uvm in that case to make that work.

Reply:
I can confirm that running these in either order works today (though doing cuml before cudf raises an innocuous warning log from cudf saying a managed memory resource is already configured.

I'm going to make some changes to how we handle managed memory and add a test for this, just noting that combining the magics works already today.

Reply:
https://github.com/rapidsai/cuml/pull/6700